rpc/src/client.rs

```diff
@@ -1,31 +1,45 @@
 //! Tendermint RPC client
 
-use bytes::buf::ext::BufExt;
-use hyper::header;
+use tendermint::{
+    abci::{self, Transaction},
+    block::Height,
+    evidence::Evidence,
+    net, Genesis,
+};
 
-use tendermint::abci::{self, Transaction};
-use tendermint::block::Height;
-use tendermint::evidence::Evidence;
-use tendermint::net;
-use tendermint::Genesis;
-
-use crate::{endpoint::*, Error, Request, Response};
+use crate::{
+    client::{
+        subscription::SubscriptionManager,
+        transport::{http_ws::HttpWsTransport, Transport},
+    },
+    endpoint::*,
+    Error, Request, Response,
+};
 
 pub mod event_listener;
+pub mod subscription;
```

Comment:
The `use` statements should be following the mod declarations, makes it easier to correlate them.